python/lsst/validate/drp/base.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+from __future__ import print_function, division
+
+
+class ValidateError(Exception):
+    """Base classes for exceptions in validate_drp."""
+    pass
```

Comment:
I don't like the idea of raising a particular exception just because the code came from a particular module; exception types should correspond to the type of error, not the home of the code.  I'd recommend reserving `ValidateError` (maybe rename it `ValidationError`?) for cases where we don't meet requirements, and use something like `RuntimeError` or `TypeError` for e.g. plotting problems or passing the wrong arguments.